packages/amplify_authenticator/example/integration_test/pages/authenticator_page.dart

```diff
@@ -99,14 +97,12 @@ abstract class AuthenticatorPage {
 
   /// Then I see User not found banner
   Future<void> expectUserNotFound() async => expectError(
-        Platform.isAndroid ? 'User not found' : 'User does not exist',
```

Comment:
Can you rebase and put these fixes into a separate commit? `git rebase -i aws/next`

Reply:
Great article which dives into this more with examples: https://github.blog/2022-06-30-write-better-commits-build-better-projects/

Reply:
Done.